src/previewCode.ts

```diff
@@ -4,9 +4,8 @@ import { ManimShell } from './manimShell';
 import { EventEmitter } from 'events';
 import { Logger } from './logger';
 
-const PREVIEW_COMMAND = `\x0C checkpoint_paste()\x1b`;
+const PREVIEW_COMMAND = `\x0Ccheckpoint_paste()`;
 // \x0C: is Ctrl + L
```

Comment:
feel free to now also move the comment on top of or to the side of the line that defines `PREVIEW_COMMAND`